README.md

```diff
@@ -1,7 +1,12 @@
 # InnerSource Patterns
 
+<a href="https://innersourcecommons.gitbook.io/innersource-patterns/">
+<img align="right" src="book/innersource-patterns-book-cover.jpg" title="The InnerSource Patterns book" width="30%"></a>
```

Comment:
Wasn't there a way to do that with Markdown too?

Reply:
I think one can do simple things like a clickable image that then leads to a URL. However I am not aware of a markdown construct that would allow us to do the `align="right"` and `width="30%"`.